plugins/auth-backend/src/providers/oauth2/provider.ts

```diff
@@ -244,6 +247,7 @@ export const createOAuth2Provider = (
       const authorizationUrl = envConfig.getString('authorizationUrl');
       const tokenUrl = envConfig.getString('tokenUrl');
       const scope = envConfig.getOptionalString('scope');
+      const basicAuth = envConfig.getOptionalBoolean('basicAuth');
```

Comment:
Perhaps `includeBasicAuth` instead? Just so it doesn't sound like you're supposed to put the actual basic auth there, and also clarify that it is included in addition the the regular method

Reply:
Fixed 👌